client/app/components/queries/ScheduleDialog.jsx

```diff
@@ -44,31 +36,31 @@ class ScheduleDialog extends React.Component {
 
   get initState() {
     const newSchedule = clone(this.props.query.schedule);
-    const { time, interval: secs, day_of_week: day } = newSchedule;
-    const interval = secs ? secondsToInterval(secs) : {};
+    const { time, interval: seconds, day_of_week: day } = newSchedule;
+    const { interval } = secondsToInterval(seconds);
     const [hour, minute] = time ? localizeTime(time).split(':') : [null, null];
 
     return {
       hour,
       minute,
-      count: interval.count ? String(interval.count) : '1',
-      interval: interval.interval || IntervalEnum.NEVER,
+      seconds,
```

Comment:
Swapped `count` for the original `seconds` since it serves as a better select option value.